### PR TITLE
[4.0] Correcting wrong debug lang untranslated strings

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes 
Adding some missing labels (filter_contacts, filter_articles), correcting a typo (filter_banners), prevent double translation for tooltip (transition-button);

Note: I could not find why `COM_NEWSFEEDS_CATEGORIES` is noted as untranslated as the sys.ini is loaded...
### Testing Instructions

**Make sure https://github.com/joomla/joomla-cms/pull/27074 is merged or merge it first.**

Set debug lang on, show its subpage and navigate in the managers concerned above
Look at the Untranslated strings.

The menutypes are noted as Untranslated but this is expected (see explanation here: https://github.com/joomla/joomla-cms/issues/27068#issuecomment-554286371 )

`Actions` is marked as double translated, but we already have an issue for that and we were not able to find a solution.

### Before patch
Example when the Articles Manager is displayed

Screen Shot 2019-11-15 at 12 21 12

### After patch

Screen Shot 2019-11-15 at 11 43 39

### CODE REVIEW

File: administrator/components/com_contact/forms/filter_contacts.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 		<field
 			name="featured"
 			type="list"
+			label="JFEATURED"
```
Comment (Quy) [inline administrator/components/com_contact/forms/filter_contacts.xml]: Look at the other filters where the format is `- Select XXXXX -`. This is only `Featured`.

**Comment by BertaOctech** [comment]
I have tested this item :white_check_mark: successfully on ac2d10c1f08cddc49692a92ee586c7e28015c229<br /><br />I followed the testing instructions and I obtained the expected result<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27078">issues.joomla.org/tracker/joomla-cms/27078</a>.</sub>

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on ac2d10c1f08cddc49692a92ee586c7e28015c229<br /><br />From #JMAD19 PBF, testing side by side with Berta, test OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27078">issues.joomla.org/tracker/joomla-cms/27078</a>.</sub>

File: administrator/components/com_contact/forms/filter_contacts.xml (hunk 1 above)
Comment (infograf768) [reply administrator/components/com_contact/forms/filter_contacts.xml]: Is it important?

Comment (Quy) [reply administrator/components/com_contact/forms/filter_contacts.xml]: For consistency. I don't know as I don't use a screenreader.

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27078">issues.joomla.org/tracker/joomla-cms/27078</a>.</sub>

**Comment by brianteeman** [comment]
Please remove RTC until https://github.com/joomla/joomla-cms/pull/27078#discussion_r346991138 is resolved

File: administrator/components/com_contact/forms/filter_contacts.xml (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_contact/forms/filter_contacts.xml]: Please change the label to JOPTION_SELECT_FEATURED

Comment (infograf768) [reply administrator/components/com_contact/forms/filter_contacts.xml]: will do

**Comment by infograf768** [comment]
Strings changed. back to RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27078">issues.joomla.org/tracker/joomla-cms/27078</a>.</sub>

**Pull request merged**